### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by hchugh08**
1.Delete list view code added
 2.created the delete button

### CODE REVIEW

**Review by ryab0 — CHANGES_REQUESTED** [review]
Reading this left me a little confused, but maybe you can clarify at tomorrow's standup. The delete functionality should be included in listView.php should it not? Also was a little confused by line 22 and 40 - would both the create and delete not be triggered everytime as they have the same if statement?

**Comment by Rolo123y** [comment]
Deleting a list should be in its own .php file which the listView.php can reference when you delete a task.

**Comment by gaurav613** [comment]
The option to delete a list would also be included in a modal(pop-up) on the listView page. Only the code for deleting can be moved to listView page.

**Comment by Rolo123y** [comment]
> eting can be moved to listView page.

**Comment by Rolo123y** [comment]
Alternatively we could have a button on listView.php which references another .php page that runs the query. This would be similar to how 'add a task' works atm.

File: deletelistView.php [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        "; 
+        $query_params = array( 
+            ':listTitle' => $_POST['name'], //Insert the user's input into the database
+            ':boardID' => 1 //Placeholder until we develop the board view function
+        ); 
+        $stmt = $db->prepare($query); 
+       	$result = $stmt->execute($query_params); 
+}
+if (!empty($_POST['name'])){
+	//If the user submitted a new delete list request we run this code
```
Comment (gaurav613) [inline deletelistView.php]: Some modifications
- This block (line 40-48) can be moved to `listView.php`

- Change the variable inside the `$_POST` to be different from that of the create list call (e.g. `name_to_delete`) - this change should also be made in the form(`<input name = "">`) for the delete task

File: deletelistView.php [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+            <label for="inputName" class="sr-only">List Name</label>
+            <input type="text" name="name" value="" class="form-control" placeholder="List Name" required/>
+            <button class="btn btn-lg btn-primary btn-block" type="submit" value="Register">Create</button>
+        </form>
+      </div>
+    </div>
+  </div>
+</div>
+<div class="modal fade" id="createList" tabindex="-1" aria-labelledby="createList" aria-hidden="true">
```
Comment (gaurav613) [inline deletelistView.php]: - This block (line 100 - 119) can also be moved to `listView.php`

- Change the form input name to the `$_POST` variable name you changed above

**Review by gaurav613 — CHANGES_REQUESTED** [review]
I have left a few comments on this:

**Comment by gaurav613** [comment]
Looks like you don't have the updated version of `listViews.php`. To fix this, pull the latest changes to your local main branch (`git pull origin main`), go to your branch(`git checkout <your-branch>`), then merge it with main (`git merge main`)

**Review by ryab0 — CHANGES_REQUESTED** [review]
@hchugh08 As discussed on our call:

- Requires a new button for Delete

- Modal should be opened by Delete button

Should use listID instead of listTitle to ensure it does not delete multiple lists with same name

**Review by ryab0 — APPROVED** [review]
Looks like you've made all the changes we discussed, nice work.

**Comment by gaurav613** [comment]
Closing now as this has been moved to #40 for cleaner commit trail/visibility of changes.

**Pull request closed without merging**